docs/dashboards/linked-filters.md

```diff
@@ -58,7 +58,7 @@ Metabase uses database column metadata to populate values for linked filters, wh
 
 - Native/SQL questions must have a [field filter](../questions/native-editor/sql-parameters.md#the-field-filter-variable-type) variable in order to be linked. Basic SQL variables aren't connected to database columns, so they won't work for linked filters.
```

Comment:
Wasn't sure about this one: it is 2 sentences, so seems right to actually end with a `.`.